pkg/config/config.go

```diff
@@ -81,3 +91,20 @@ func validateAdditionalResources(resources []string) error {
 	}
 	return nil
 }
+
+// validateOutputFile checks if output file name is valid and if the
```

Comment:
do we need the comments? I feel the name of the function is suffiecent

Reply:
Comments are standard for generating docs using `godoc`. And while it typically does not display unexported funcs, it does have a switch to do that. So for that reason, I would prefer to have, hopefully meaningful, comments as a standard.

I agree though that these are not very helpful, so would be happy for better suggestions :).

Reply:
ah, if they are generated, then we should like generate them for the whole repo. Rather than just these changes.

Reply:
I don't mean the comments themselves are generated, but that these are used in generated documentation. E.g. a few times mentions [^1] is generated from [^2]. I agree that at some time I would like to have comments for all funcs, probably when we put in some linter to check for that, but for now, I would prefer to add those rather than not.


[^1]: https://pkg.go.dev/io#Closer
[^2]: https://cs.opensource.google/go/go/+/refs/tags/go1.19.1:src/io/io.go;l=100-106

Reply:
That btw. brings up a very good question - we do build on many platforms, incl. Windows. Should we run tests on all of those as well?

 _I would say yes, but not sure how difficult is the implementation._